From e07067f57645db870281f2a9b2915d4eefe97982 Mon Sep 17 00:00:00 2001
From: Hari Subramaniyan <harisankar@apache.org>
Date: Wed, 31 Aug 2016 11:34:23 -0700
Subject: [PATCH 1017/1164] CDH-48309 HIVE-14670:
 org.apache.hadoop.hive.ql.TestMTQueries failure
 (Hari Subramaniyan, reviewed by Prasanth
 Jayachandran)

(cherry picked from commit fa3a8b9bd16aad66b869ee30f6ddfa12f2fd5ad1)

Change-Id: I9f58400c87b1c06008c7e714fb4a8fe33bfb6896
---
 .../org/apache/hadoop/hive/ql/TestMTQueries.java   |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMTQueries.java b/itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMTQueries.java
index 95691f6..5ce4118 100644
--- a/itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMTQueries.java
+++ b/itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/TestMTQueries.java
@@ -33,8 +33,7 @@ public TestMTQueries() {
   }
 
   public void testMTQueries1() throws Exception {
-    String[] testNames = new String[] {"join1.q", "join2.q", "groupby1.q",
-        "groupby2.q", "join3.q", "input1.q", "input19.q"};
+    String[] testNames = new String[] {"join2.q", "groupby1.q", "input1.q", "input19.q"};
 
     File[] qfiles = setupQFiles(testNames);
     QTestUtil[] qts = QTestUtil.queryListRunnerSetup(qfiles, resDir, logDir);
-- 
1.7.9.5

